src/components/pages/NotFoundErrorPage.tsx

```diff
@@ -23,14 +20,27 @@ const NoteFoundErrorPage = ({
               style={{ marginRight: 10, marginLeft: 15 }}
               size={24}
             />
-            <span style={{ marginRight: 10 }}>{title}</span>
+            <span style={{ marginRight: 10 }}>Page Not Found</span>
           </>
         ),
       }}
     >
-      <div style={{ marginLeft: 15 }}>{description}</div>
+      <div style={{ marginLeft: 15 }}>
+        <p>Choose other space or create a new one</p>
```

Comment:
I am not sure the description fits. 
The button seems to either login or go to space creation.

Reply:
This page is for local space routing so users won't see this if they don't use local spaces

Reply:
Also, we will discard local routing completely later so I think we can ignore it for now.